### DISCUSSION TITLE
Battery Drain after charging

### DISCUSSION THREAD

**Discussion by delacroix17** [rg35xx-cfw/Koriki#84] [General]
I was wondering if this was fixed. I remember that if you charged it and unplugged it, it would still act like it was on and lose power unless you booted up and selected "Shutdown." Now I know in 1.0 that if you remove it from charging it goes through a shutting down process, but I think it's still losing power. Can anyone confirm this was fixed?

**Comment by benni092** [comment]
There is definitely something wrong. Plug in power, Koriki starts to boot and shows the battery-symbol. Plug out power, Koriki shows sutting down and it seems it is shut down. But when i plug the power in, it doesn't boot anymore. Pressing the power button also do nothing. I have to press the reset button to make it working again.

**Reply by acmeplus** [reply]
Yes, once it's full shutoff you need to either press reset or long press power to boot. That's expected. The console in that state is fully off, so that's why you need to do that.

**Comment by acmeplus** [comment]
> I was wondering if this was fixed. I remember that if you charged it and unplugged it, it would still act like it was on and lose power unless you booted up and selected "Shutdown." Now I know in 1.0 that if you remove it from charging it goes through a shutting down process, but I think it's still losing power. Can anyone confirm this was fixed?

In my tests the console was fully off, and battery was more or less at the same level after a while. Granted that I didn't have time to do long term tests.

**Reply by korikirg35xx** [reply]
With 1.0.1 version i loose power when the device is full shutdown (from menu and after charging)
